### ISSUE TITLE
Interpolate direction

### ISSUE THREAD

**Issue by nomeata** [movingpandas/movingpandas#359]
#### Is your feature request related to a problem?

I wish I could use MovingPandas to analyze paragliding tracks, and I am mostly looking for a nice way to interpolate the tracks and query various parameters at certain times.

#### Describe the solution you'd like

I see there is `add_direction` which adds the direction between adjacent points. But for working with an interpolated track (e.g. using `get_position_at`), I’d also expect to be able to run `get_direction_at` with a suitable interpolation.

**Comment by nomeata** [comment]
Upon further thought, I notice that I am not sure whether movingpandas even performs interpolations (like splines) where the direction changes between adjacent points, so maybe the request is pointless because it’s only linearly interpolating?

**Comment by anitagraser** [comment]
Indeed, MovingPandas performs linear interpolation and there is no spline interpolation so far.

**Comment by nomeata** [comment]
Thanks, and sorry for the incomplete research. Feel free to close or keep open as it's more useful to you. For my purposes I'll play with https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.splprep.html and see if that works.

**Comment by anitagraser** [comment]
Maybe the kalman smoother could help: https://movingpandas.readthedocs.io/en/main/trajectorysmoother.html
